# HG changeset patch
# User Rik <rik@octave.org>
# Date 1379549492 25200
#      Wed Sep 18 17:11:32 2013 -0700
# Node ID 47269b03a9463db286cc8aba2b290bcc81909a3a
# Parent  4a6a4657fdf285ba0958adce597b0f8beb8fd28a
__next_line_color__.m: Remove use of CamelCase.

* scripts/plot/private/__next_line_color__.m: Remove use of CamelCase.

diff --git a/scripts/plot/private/__next_line_color__.m b/scripts/plot/private/__next_line_color__.m
--- a/scripts/plot/private/__next_line_color__.m
+++ b/scripts/plot/private/__next_line_color__.m
@@ -32,23 +32,23 @@ function rgb = __next_line_color__ (rese
   persistent reset_colors = true;
 
   if (nargin == 1)
     ## Indicates whether the next call will increment or not
     reset_colors = reset;
   else
     ## Find and return the next line color
     ca = gca ();
-    colorOrder = get (ca, "ColorOrder");
+    colororder = get (ca, "colororder");
     if (reset_colors)
       color_index = 1;
       reset_colors = false;
     else
       ## Executed when "hold all" is active
-      nChildren = length (get (ca, "Children"));
-      nColors = rows (colorOrder);
-      color_index = mod (nChildren, nColors) + 1;
+      n_kids = length (get (ca, "children"));
+      n_colors = rows (colororder);
+      color_index = mod (n_kids, n_colors) + 1;
     endif
-    rgb = colorOrder(color_index,:);
+    rgb = colororder(color_index,:);
   endif
 
 endfunction
 
